ruby/red-arrow-format/lib/arrow-format/array.rb

```diff
@@ -110,13 +110,16 @@ def to_a
     end
   end
 
-  class DateArray < Array
+  class TemporaryArray < Array
```

Comment:
Just interest. `Temporary` means temorary? (It mean this class name will change later)